syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md

```diff
@@ -0,0 +1,573 @@
+
+---
+
+## “The best blockchain today will not be the best blockchain tomorrow.”
+
+Notes:
+
+This philosophy is a realization that building a blockchain is trying to build a piece of software which will last forever. The we will not be successful engineering Polkadot simply for the problems we find today. By the time we have built X, the world will need Y, and so on. This is why we have invested so much time building platforms and SDKs, not just products. We need to make sure these technologies are able to adapt and evolve in order to stay relevant for users
+
+---
+
+## The Goals of Polkadot
+
+---
+
+## The Blockchain Scalability Trilemma
+
+<div class="grid grid-cols-2">
+
+<div>
+
+<img style="height: 400px;" src="../../assets/img/7-Polkadot/trilemma.svg" />
+
+</div>
+
+<div>
+
+- Security: How much does it cost to attack the network?
+
+- Scalability: How much work can the network do?
+
+- Decentralization: How decentralized is the network?
+
+</div>
+
+</div>
+
+---
+
+## In one sentence...
+
+> Polkadot’s mission is to provide secure, scalable, and resilient infrastructure for Web3 applications and services.
+
+Notes:
+
+Note the difference between “decentralization” as a mission vs "resilience".
+
+---
+
+Polkadot tries to accomplish that mission by solving three problems:
+
+<pba-flex center>
+
+1. Computational Scalability
+1. Shared Security
+1. Interoperability
+
+</pba-flex>
+
+---
+
+## The Decisions
+
+What are the type 1 decisions which define Polkadot?
+
+---
+
+## Wasm
+
+<div class="grid grid-cols-3">
+
+<div>
+
+<img style="width: 400px;" src="../../assets/img/0-Shared/logo/webassembly-blue.png" />
+
+</div>
+
+<div class="col-span-2">
+
+WebAssembly is the backbone of Polkadot. It is a fast, safe, and open meta-protocol which powers all of the state transitions of our ecosystem.
+
+It standardizes how chains execute, sandboxes that execution for improved security, and allows teams to build on Polkadot using any language that can be compiled into Wasm.
+
+</div>
+
+</div>
+
+---
+
+## Sharding
+
+<div class="grid grid-cols-2">
+
+<div>
+
+Polkadot scales primarily by parallelizing execution on separate data shards.
+
+These parallel chains (shards) are called Parachains.
+
+</div>
+
+<div>
+
+<img style="width: 500px;" src="../../assets/img/7-Polkadot/decisions/sharding.svg" />
+
+</div>
+
+</div>
+
+---
+
+## App-Chains
+
+<div class="grid grid-cols-2">
+
+<div>
+
+<img style="width: 500px;" src="../../assets/img/7-Polkadot/decisions/app-chain.svg" />
+
+</div>
+
+<div>
+
+Another key scaling decision is the choice of heterogeneous shards, allowing for application specific chains.
+
+Specialized solutions for problems are more performant than generalized solutions, as they can incorporate more details about the problem space.
```

Comment:
App-chains are genereally understood as individual parachains that are a full application by themselves. Seems that this mindset is changing and multi-chain applications are being more favoured. For instance assethub is meant to be used by most parachains for all things tokens.